image/templates/helm/stackrox-central/templates/_init.tpl.htpl

```diff
@@ -64,6 +64,15 @@
 {{ end }}
 [< end >]
 
+{{ if $._rox.central.db.external }}
+  {{ if not $._rox.central.db.source.connectionString }}
+    {{ include "srox.warn" (list $ "You have chosen to bring your own Central DB without providing its connection string. We are using the default source string. To ensure the connection to your Central DB, you may override it with `--set central.db.source.connectionString=<ConnectionString>`.") }}
```

Comment:
Is this possible to happen in operator? What happens if we warn within operator install?